test/e2e/basic_integration_test.go

```diff
@@ -16,7 +16,7 @@ var _ = Describe("When testing basic cluster creation [basic]", Label("basic"),
 		validateGlobals(specName)
 
 		// We need to override clusterctl apply log folder to avoid getting our credentials exposed.
-		clusterctlLogFolder = filepath.Join(os.TempDir(), "clusters", bootstrapClusterProxy.GetName())
+		clusterctlLogFolder = filepath.Join(os.TempDir(), "target_cluster_logs", bootstrapClusterProxy.GetName())
```

Comment:
It should be `target_clusters_logs`, if there could be more than 1 target cluster. Or we can omit `logs` (which makes it `target_clusters`), as it's clearly a log directory.

And why do we include bootstrap cluster name here? Shouldn't it be the target cluster name?

Reply:
Project Infra checks only this directory and copies whatever is in it. We will need to make changes there too, I can open a separate PR which will just be fixing the naming of these folders in capm3 and project-infra.